dashboard/test/models/lesson_test.rb

```diff
@@ -225,6 +225,28 @@ class LessonTest < ActiveSupport::TestCase
     assert levels_data.last[:bonus]
   end
 
+  test 'summarize uses unplugged property if the lesson is migrated' do
+    script = create :script, is_migrated: true, hidden: true
+    lesson_group = create :lesson_group, script: script
+    lesson = create :lesson, lesson_group: lesson_group, script: script, name: 'Lesson 1', key: 'lesson-1', relative_position: 1, absolute_position: 1, unplugged: true
+
+    levels_data = lesson.summarize
+    assert levels_data[:unplugged]
+  end
+
+  test 'summarize uses display_as_unplugged if the lesson is not migrated' do
```

Comment:
nit: I don't know that this method checks that we use display_as_unplugged so much as it checks we are not using the unplugged property. That is still a valuable thing to check and probably the thing we should be checking since there should probably be something else testing the display_as_unplugged logic already so I'm only suggesting that it might be more clear to update this name